### ISSUE TITLE
We just moved to the reactjs Github org

### ISSUE THREAD

**Issue by gaearon** [reactjs/redux#1392]
Hi folks. (TL;DR is at the bottom.)

We have moved Redux, React Router and other community projects over to [reactjs](https://github.com/reactjs) Github org.

We did this because in the future, we want to ensure Redux and related projects don’t end up like Express. We would like a separate legal entity (such as a foundation) to own that code. This is a work in progress and I can’t give you exact details but ensuring those libraries are well cared for and don’t end up as some company’s property (including Facebook itself) is the goal.

You might have noticed that [reactjs](http://github.com/reactjs) has never been the home of React itself. React lives under the [facebook](http://github.com/facebook) org because it is Facebook’s project.

**On the other hand, [reactjs](http://github.com/reactjs) was meant for community projects such as [react-rails](https://github.com/reactjs/react-rails).** I think we can all agree that React Router, Redux, and a few other Rackt libraries deserve to be there, as they are most widely used together with React. In January 2016, every second `npm install react` also included `react-router`, and every third `npm install react` also included Redux.

We want [reactjs](http://github.com/reactjs) to be the home for projects that are widely depended upon in the React ecosystem, whether or not they depend on React itself. This is why Redux, while technically independent on React, ends up there. The examples and the documentation have always been oriented at React users, so it shouldn’t come as a big surprise that we want to be closer to the React community.

**In no way will this impact the technical direction of Redux core**. We are not going to suddenly force you to sign up for Facebook or introduce a React dependency to the core. It is sufficient to read the source code for the core to understand that this wouldn’t make any sense. For example, we are enthusiastic about Redux [being adopted by the Angular community](https://medium.com/google-developer-experts/angular-2-introduction-to-redux-1cf18af27e6e) as well as by [Angular-specific takes on Redux](https://github.com/ngrx/store).

This move was made because @ryanflorence, @mjackson, me, and other contributors wanted to ensure that Rackt family of projects doesn’t end up as IP of some single company. We are talking to Facebook, and they might help us through the legal hurdles of ensuring this.

Finally, all collaborators on the Rackt team should have received their invitations to the [reactjs](http://github.com/reactjs) org. If we messed up the permissions and something doesn’t work that used to work, please ping me in DM on Twitter or via email, and we’ll make this right.

### TL;DR

* [rackt](http://github.com/rackt) is what [reactjs](http://github.com/reactjs) org should have been, so we decided to merge
* We moved Redux to [reactjs](http://github.com/reactjs) org
* Redux is not and will not be owned by Facebook
* This is about community, not tech
* Redux stays independent of React

**Comment by STRML** [comment]
This is a solid move for the future of Redux and the React ecosystem. Thanks @gaearon for all your hard work, and I can confidently say we're all looking forward to the future!

**Comment by garbles** [comment]
:tada:

**Comment by auvipy** [comment]
awesome!!!

**Comment by chicoxyzzy** [comment]
what about Reselect?

**Comment by gaearon** [comment]
We haven’t talked to the authors yet so I haven’t moved it. Will be up to them.

**Comment by NogsMPLS** [comment]
great move!

**Comment by jakedahm** [comment]
+1

**Comment by mjackson** [comment]
> what about Reselect?

https://github.com/rackt/reselect/issues/81

**Comment by jquense** [comment]
seems like a great idea... I don't quite understand though how moving github orgs consistituets any sort of legal safety though.  each project is still bound by their chosen license.

also the intent twinges a bit when the organization you moved too is managed and owned by Facebook. are there any concert plans for governance?

**Comment by rossipedia** [comment]
> we want to ensure Redux and related projects don’t end up like Express.

Ummm... what did I miss?

**Comment by tngdvd** [comment]
@rossipedia Express was owned by Strongloop then Strongloop was acquired by IBM. Everyone wondered about the future of Express, but the Node foundation said they'll bring the project in house.

**Comment by gaearon** [comment]
>seems like a great idea... I don't quite understand though how moving github orgs consistituets any sort of legal safety though. each project is still bound by their chosen license.

By itself, it doesn’t mean anything other than React team recognizing these projects as important and giving them spotlight. There are plans to turn this into an independent legal entity but I can’t say more details at this point.

>also the intent twinges a bit when the organization you moved too is managed and owned by Facebook. are there any concert plans for governance?

The `routing` and `redux` teams inside the `reactjs` organization are managed by us: Ryan, Michael, me, and all the previous maintainers. Whoever was admin on the old teams, is also an admin on the new teams. Facebook has zero intention of messing with our repos, and as I said earlier, the plan is to make this `reactjs` organization independent of Facebook itself.

>Ummm... what did I miss?

AFAIK Express was owned by a single person, later bought by a company. I just learned it [is going to be part of Node Foundation](http://www.infoworld.com/article/3031686/javascript/nodejs-foundation-to-shepherd-express-web-framework.html) but we want to avoid the whole debacle :smile: . The move to the Github org is only a start, as I said above.

**Comment by nalwayaabhishek** [comment]
great :+1:

**Comment by andrewdamelio** [comment]
:tada:

**Comment by tychota** [comment]
@gaearon not a concern but more a need of more information. Not a critique. But ...

While I agree on the need to act and to move to some foundation,  I don't understand how reactjs can help. Why not apply here https://github.com/nodejs/TSC/tree/master/Applications ? Node.js seems to have a lot of experience in term of foundation, and is backed up with Linux one (hope I said nothing wrong) !

**Comment by gaearon** [comment]
The move to http://github.com/reactjs is motivated by the fact that:

1. React beginners suffer from choice paralysis and aren’t sure which packages are tested in production
2. “rackt” was always a weird name to begin with, and its positioning has always been “Quality React code”
3. React team is interested in having quality contributions in a single place, maintained by the community

We don’t know the legal details yet. We are talking about this, and these talks take time. We’ll let everyone know when/if we announce a foundation or something else.

In the meantime, think of http://github.com/reactjs as of a rebranding of http://github.com/rackt with React team’s approval but with the original maintainers and project owners in full control. It’s really nothing more than that.

And it’s definitely not going to make “more React oriented”, “owned by Facebook” or anything. These concerns are baseless. If there is ever any intention of this (which there is not), I’ll transfer Redux to my personal account. I am still its author :wink: .

**Comment by colinhoernig** [comment]
I agree with this move wholeheartedly. Merging the Rackt family of projects to the reactjs org is great for the future of the React (..and JS) ecosystem, provided the legal aspects follow through as expected.  I appreciate what @gaearon, @mjackson, @ryanflorence, and others are hoping to achieve.  If Facebook does indeed help with the legal aspects of making sure the Rackt projects do not become IP of any one company, but instead are controlled by something such as a foundation, that would be quite excellent.  Best of luck!

**Comment by BerkeleyTrue** [comment]
@gaearon How about moving [warning](https://github.com/r3dm/warning) to this org? It seems like it would be quite at home here.

**Comment by gaearon** [comment]
I think we want modules there to be useful to React users and not just library authors. Modules like `invariant` and `warning` say more about Facebook’s internal conventions rather than community’s needs. I don’t think they are an important part of using React.

**Comment by BerkeleyTrue** [comment]
Fair enough

**Comment by tonyspiro** [comment]
great news :tada: :beers: !

**Comment by kumavis** [comment]
dig it

**Comment by StevenNguyenHung** [comment]
really love the idea. Hope we will see more contributions to the reactjs family. thanks @gaearon for all the hard work (y)

**Comment by ryanflorence** [comment]
> also the intent twinges a bit when the organization you moved too is managed and owned by Facebook. are there any concert plans for governance?

I'm going to directly quote @tomocchino as we started this:

> I mean like.. we can make you guys admins of the github.com/reactjs, and you can have it

**Comment by barrystaes** [comment]
:tada: Glad to see that https://github.com/reactjs/react-router-redux also moved. :family:

**Comment by vanthiyathevan** [comment]
This is a great initiative. Would like to see [recompose](https://github.com/acdlite/recompose/) and [redux-actions](https://github.com/acdlite/redux-actions) under this org as well. Both of these libraries are awesome for writing functional style react/redux apps.
